mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/clients/amtenhetsregister/AmtEnhetsregisterClient.kt

```diff
@@ -0,0 +1,7 @@
+package no.nav.mulighetsrommet.api.clients.amtenhetsregister
```

Comment:
Kan evt. kalle pakkenavnet for `amt_enhetsregister` for å gjøre det litt mer leslig? Tilsvarende for `arena_ords_proxy`?